### ISSUE TITLE
Earth Foundation Models

### ISSUE THREAD

**Issue by nagataka** [nagataka/Read-a-Paper#58]
Curated with ChatGPT:

以下の五つのカテゴリで整理する。

A. Satellite foundation models
B. Self-supervised remote sensing
C. Weather / climate AI
D. Spatiotemporal ML
E. Multimodal Earth models

#### [SatMAE: Pre-training Transformers for Temporal and Multi-Spectral Satellite Imagery](https://sustainlab-group.github.io/SatMAE/)

以下を学ぶ
multispectral representation
self-supervised satellite pretraining

#### [A Scale-Aware Masked Autoencoder for Multiscale Geospatial Representation Learning](https://ai-climate.berkeley.edu/scale-mae-website/)

以下を学ぶ
scale invariance
satellite spatial resolution

#### [GMF](Towards Geospatial Foundation Models via Continual Pretraining)

Earth embedding を作るモデル

#### [Masked Autoencoders Are Scalable Vision Learners](https://arxiv.org/abs/2111.06377)

MAEの原論文。

#### [DINOv2]

- 
- [AlphaEarth Foundations](https://deepmind.google/blog/alphaearth-foundations-helps-map-our-planet-in-unprecedented-detail/)
